evaluation/cli.py

```diff
@@ -48,7 +51,30 @@ def evaluate(
         exists=True,
         dir_okay=False,
         file_okay=True,
-        help="Path to the configuration JSON file.",
+        help=f"Path to the configuration JSON file. The name of the model to be evaluated should be specified in the JSON file. The available models that you can choose from are: {', '.join(get_models(get_model_url))}",
+        default=DEFAULT_CONFIG_PATH,
+    ),
+    numquestions: Optional[int] = typer.Option(
+        help="Number of questions to evaluate (defaults to all if not specified).",
+        default=None,
+        parser=int_or_none,
+    ),
+    targeturl: Optional[str] = typer.Option(
+        help="URL of the target service to evaluate (defaults to the value of the BACKEND_URI environment variable).",
+        default=None,
+        parser=str_or_none,
+    ),
+):
+    run_evaluation_from_config(EVALUATION_DIR, load_config(config), numquestions, targeturl)
+
+
+@app.command()
+def compare(
+    config: Path = typer.Option(
+        exists=True,
+        dir_okay=False,
+        file_okay=True,
+        help=f"Path to the configuration JSON file.The name of the models to be compared should be specified in the JSON file. The available models that you can choose from are: {', '.join(get_models(get_model_url))}",
```

Comment:
> What does this function do? It looks the same as the evaluate function except for the function name.

Yeah they are pretty similar because they both enter the same entrance function, the only difference is the command name. I just wanted to show that we have both evaluate and compare functions and have pointed that out in the readme. But yeah they are kind of redundant and I'm thinking about combining these two into a single command. What do you think @phoevos